test/conformance/testdata/conformance.yaml

```diff
@@ -811,7 +811,7 @@
     new message, the check MUST have the update message. The event is deleted and
     MUST NOT show up when listing all events.
   release: v1.19
-  file: test/e2e/framework/events/events.go
+  file: test/e2e/apimachinery/events.go
```

Comment:
@johnbelamaric this is why we need your /approve for a PR that's just adding a new non-conformance test, @oomichi caught that we shouldn't have added a conformance test to the test/e2e/framework directory (ref: https://github.com/kubernetes/kubernetes/pull/91230#discussion_r430713176) so it's been moved to the correct file

Reply:
oops. ok. has it been running from there though? I mean, do we have soak time for this?

Reply:
i.e., it is added as a conformance test below. so I want to make sure it was actually running at some point as a non-conformance test. i can check the board